docs/maps/maps-getting-started.asciidoc

```diff
@@ -0,0 +1,183 @@
+== Getting started with Maps
+
+*Maps* is your place for analyzing geospatial data. For example, a map might 
+show the origin, destination,
+and average time of flights or web log requests by country.
+
+You work with *Maps* by adding layers. The data for a layer can come from 
+sources such as {es} documents, vector sources, tile map services, web map 
+services, and more. You can symbolize the data in different ways. 
+For example, you might show which airports have the longest flight 
+delays by using circles from small to big. Or, 
+you might show the amount of web log traffic by shading countries from 
+light to dark. 
+
+[role="screenshot"]
+image::maps/images/sample_data_web_logs.png[]
+
+Your best bet to getting started with *Maps* is to add one of 
+the sample data sets that ship with {kib}. Adding a sample data set also 
+loads a map to go with the data.
+
+. In the upper left corner of your browser, click the {kib} logo to navigate
+to the {kib} home page.
+. Click *Load a data set and a {kib} dashboard*.
+. Add the *Sample web logs* data set.
+
+. In the side navigation, click *Maps*. 
+
+. In the maps list, click *[Logs] Total Requests and Bytes*.
++
+This loads the map for the web logs data set. Check out each of the three layers of the map. As you move your mouse over the map,
+note how the tooltips show property values. 
+
+You'll re-create the *[Logs] Total Requests and Bytes* map in this tutorial. 
+You'll learn how to:
+
+* Create a multi-layer map
+* Connect a layer to a data source
+* Use symbols, colors, and labels to style a layer
+* Create layers for {es} data
+
+
+=== Creating a new map
+
+The first thing to do is to create a new map.
+
+. In the application header, click *Maps* to return to the list of maps.
+. Click *Create map*.
++
+A new map is created using a base tile layer.
++
+[role="screenshot"]
+image::maps/images/gs_create_new_map.png[]
+
+
+=== Adding a choropleth layer
+
+Now that you have a map, you'll want to add layers to it.
+The first layer you'll add is a choropleth layer to shade world countries 
+by web log traffic. Darker shades symbolize countries with more web log traffic, 
+and lighter shades symbolize countries with less traffic.
+
+==== Add a vector layer from the Elastic Maps Service source
+
+. In the map legend, click *Add layer*.
+. Click the *Vector shapes* data source.
+. From the *Layer* dropdown menu, select *World Countries*.
+. Click the *Add layer* button.
+. Set *Layer name* to `Total Requests by Country`.
+. Set *Layer transparency* to 0.5.
+
+===== Join the vector layer with the {es} index
+
+You must add the web log traffic property to the world countries so
+that the property is available for styling.
+You'll create a <<terms-join, terms join>> to link the vector source *World Countries* to 
+the {es} index `kibana_sample_data_logs` on the shared key iso2 = geo.src. 
+
+. Click plus image:maps/images/gs_plus_icon.png[] to the right of *Term Joins* label.
```

Comment:
Showing the plus icon is a really nice touch. Thanks for adding this

Reply:
I like the update. Much better. [add-sample-data](https://github.com/elastic/kibana/blob/master/docs/getting-started/add-sample-data.asciidoc) has been merged in master. Just pull the latest into your PR with the following commands

```
git fetch upstream
git pull upstream master
git push origin docs/maps-tutorial
```